src/modules/users/services/user.service.ts

```diff
@@ -39,11 +39,11 @@ export class UserService {
   }
 
   async checkIfExists (email: string): Promise<void> {
-    const exists = await this.userRepository.findOne({
+    const exists = await this.userRepository.exist({
```

Comment:
```suggestion
    const exists = await this.userRepository.exists({
```

Normaal is exist deprecated in Typeorm en moet je exists gebruiken?